managed/services/agents/mysql.go

```diff
@@ -16,28 +16,36 @@
 package agents
 
 import (
+	"bytes"
 	"fmt"
 	"sort"
+	"text/template"
 	"time"
 
+	"github.com/AlekSi/pointer"
+	"github.com/pkg/errors"
+
 	agentv1 "github.com/percona/pmm/api/agent/v1"
 	inventoryv1 "github.com/percona/pmm/api/inventory/v1"
 	"github.com/percona/pmm/managed/models"
 	"github.com/percona/pmm/managed/utils/collectors"
 	"github.com/percona/pmm/version"
 )
 
-var mysqlExporterVersionWithPluginCollector = version.MustParse("2.36.0-0")
-
-// The mysqldExporterConfig returns desired configuration of mysqld_exporter process.
+var (
+	mysqlExporterVersionWithPluginCollector = version.MustParse("2.36.0-0")
+	// TODO: put back 3.2.0 when 3.1.0 is released.
+	mysqlExporterMySQL8Support = version.MustParse("3.1.0-0")
+)
```

Comment:
can we move this to version package, so everything related to versions is there.

Reply:
Done!